### ISSUE TITLE
Some utf characters in python scripts cant be loaded

### ISSUE THREAD

**Issue by skydrome** [hexchat/hexchat#2634]
Windows 10 x64 v2.16.0
```python3
import hexchat

__module_name__ = '1'
__module_author__ = '1'
__module_version__ = '1'
__module_description__ = '1'

print("this icon doesnt render properly: 📰")
print("this character doesnt render either: █") # U+2588
print("this causes an error: ═") # U+2550
```

output:
```
 this icon doesnt render properly: ðŸ“°
 this character doesnt render either: â–ˆ
```

and with that last character:
```
 Failed to load module: 'charmap' codec can't decode byte 0x90 in position 371: character maps to <undefined>
 Traceback (most recent call last):
   File "<init code for '_hexchat_embedded'>", line 150, in loadfile
   File "C:\Program Files\Python36\Lib\encodings\cp1252.py", line 23, in decode
     return codecs.charmap_decode(input,self.errors,decoding_table)[0]
 UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 371: character maps to <undefined>
```

https://github.com/hexchat/hexchat/blob/333a02d015fcea22a7ac5aaec086ec17b8bb195d/plugins/python/python.py#L149
seems you may need to explicitly open the file with `encoding="utf-8"`? else it defaults to cp1252

**Comment by TingPing** [comment]
Please test a new build here: https://github.com/hexchat/hexchat/actions/runs/1298456511

**Comment by skydrome** [comment]
this works, thanks

**Comment by TingPing** [comment]
Fixed by 6fd8a8f9bf48562fbbc42f908321b67d436c097e
